hack/boilerplate/boilerplate.py

```diff
@@ -155,7 +155,7 @@ def file_extension(filename):
     return os.path.splitext(filename)[1].split(".")[-1].lower()
 
 
-skipped_dirs = ['Godeps', 'third_party', '_gopath', '_output', '.git', 'cluster/env.sh',
```

Comment:
What does this change have to do with the rest?

Reply:
No need Godeps any more.

Reply:
No need Godeps any more.